js/schedule.js

```diff
@@ -347,13 +370,28 @@ function Schedule(CUSTOM_CONFIG) {
     $('#session-detail-wrapper').remove();
   }
 
+  schedule.saveScrollState = function(page, scrollY) {
+    var scrollStates = JSON.parse(localStorage.getItem('scrollStates')) || {};
+    scrollStates[page] = scrollY;
+    localStorage.scrollStates = JSON.stringify(scrollStates);
+  }
+
+  schedule.loadScrollState = function(page) {
+    if(schedule.previous === window.location.hash){
```

Comment:
Let's wrap this in `if (Modernizr.localstorage) {}` so the app doesn't break when `localStorage` is not available. 😁